libs/dh/message-archive/feature-log-search/src/lib/dh-message-archive-log-search.component.ts

```diff
@@ -61,62 +86,95 @@ import { DhMessageArchiveLogSearchResultScam } from './searchresult/dh-message-a
   providers: [
     DhMessageArchiveDataAccessApiStore,
     DhMessageArchiveDataAccessBlobApiStore,
+    DhMessageArchiveActorDataAccessApiStore,
   ],
+  encapsulation: ViewEncapsulation.None,
 })
-export class DhMessageArchiveLogSearchComponent implements OnDestroy {
-  private destroy$ = new Subject<void>();
-  private regexLogNameWithDateFolder = new RegExp(
-    /\d{4}-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])\/.*/
-  );
-  private regexLogNameIsSingleGuid = new RegExp(
-    /[\da-zA-Z]{8}-([\da-zA-Z]{4}-){3}[\da-zA-Z]{12}$/
-  );
+export class DhMessageArchiveLogSearchComponent {
+  searchForm: FormGroup = new FormGroup({
+    messageId: new FormControl(''),
+    rsmNames: new FormControl([]),
+    processTypes: new FormControl([]),
+    senderId: new FormControl(''),
+    receiverId: new FormControl(''),
+    includeRelated: new FormControl<boolean>({ value: false, disabled: true }),
+    dateRange: new FormControl<WattRange>(
+      {
+        start: '',
+        end: '',
+      },
+      [WattRangeValidators.required()]
+    ),
+    timeRange: new FormControl<WattRange>({
+      start: '00:00',
+      end: '23:59',
+      disabled: true,
+    }),
+  });
 
   searchResult$ = this.store.searchResult$;
   searching$ = this.store.isSearching$;
   hasSearchError$ = this.store.hasGeneralError$;
   continuationToken$ = this.store.continuationToken$;
+  isInit$ = this.store.isInit$;
+
+  localTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
suggestion: we've been using `Europe/Copenhagen` timezone in some places throughout the app already. Let's use it here as well.

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1178/commits/23df9afdac5a94e4c669c23aac3d1142045531d9